dGame/EntityManager.cpp

```diff
@@ -166,7 +167,7 @@ void EntityManager::DestroyEntity(Entity* entity) {
 
 void EntityManager::SerializeEntities() {
 	for (int32_t i = 0; i < m_EntitiesToSerialize.size(); i++) {
-		const LWOOBJID toSerialize = m_EntitiesToSerialize.at(i);
+		const LWOOBJID toSerialize = m_EntitiesToSerialize[i];
```

Comment:
> haven't we been preferring `.at(i)` instead of `[i]` due to bounds access? or is this no longer a concern due to other changes?

I've loosened a bit in regards to that on my end; if the bounds check is within a screen length, then using subscripting is ok, but if you can't see it on your screen, then the index should be assumed to be unsafe (i.e. a large loop).

As far as range based in this context it is preferred to use a method that has zero invalidation as if in the future, logic in this code path is added which can add to this container while we are iterating it, the iterators will be invalidated.  Being a core location in the codebase, I am choosing to keep the 100% safe iterator free version as opposed to a nicer reading one.